src/components/list/list.jsx

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    children: React.PropTypes.any,
+    type: React.PropTypes.oneOf(['ordered', 'unordered', 'description', 'icon'])
+  },
+
+  render: function() {
+    if (this.props.type === 'ordered') {
+      return (
+        <ol className="component-ordered-list">
+          {this.props.children}
```

Comment:
We should restrict here so you can only pass children that are of type `li` / `dt/dd` etc for each type. eg: `React.PropTypes.instanceOf(ListItem)`